tests/mssql/test_mssql.py

```diff
@@ -87,6 +96,37 @@ def test_connection_params():
     }
 
 
+def test_connection_params_with_trusted_server_certificate():
+    connector = MSSQLConnector(name="my_mssql_con", host="myhost", user="myuser", trust_server_certificate=True)
+    assert connector.get_connection_params(None) == {
+        "driver": "{ODBC Driver 18 for SQL Server}",
+        "server": "myhost",
+        "user": "myuser",
+        "as_dict": True,
+        "TrustServerCertificate": "yes",
+    }
+    connector = MSSQLConnector(
+        name="my_mssql_con",
+        host="myhost",
+        user="myuser",
+        password="mypass",
+        port=123,
```

Comment:
```suggestion
        port=12345,
```

nit: that's a root port, let's use a config we could encounter IRL :slightly_smiling_face: